### ISSUE TITLE
MySQL Backup does not work when using sockets.

### ISSUE THREAD

**Issue by spaceemotion** [freekmurze/laravel-backup#24]
Today, I discovered that the scheduled backups I had on a client's project weren't working quite as intended. Digging through the files, I found the cause of the problem: 
```
mysqldump: Got error: 2002: Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2) when trying to connect
```

It seems like the backup script does not pick up the socket configuration I have in my `config/database.php`:

```php
['unix_socket' => '/tmp/mysql5.sock']
```

Fixing this would be much appreciated! Thanks for this wonderful package :)

**Edit:** Maybe this helps: http://stackoverflow.com/questions/15318875/mysqldump-cannot-connect-using-socket

**Comment by freekmurze** [comment]
This is solved in version 2.2.0

**Comment by spaceemotion** [comment]
Thank you very much!
